source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -86,17 +85,17 @@ public ActionResult PeriodEvaluation(string period)
                 return new HttpStatusCodeResult(403, "Access Denied");
             }
 
-            var isEvaluated = evaluation.Any(p => p.UserName == loggedUser);
-            var isEvaluator = evaluation.Any(p => p.ResponsibleId == loggedUser || (p.Evaluators != null && p.Evaluators.Contains(loggedUser)));
+            var userPeriods = RavenSession
```

Comment:
Since this query is duplicated, maybe you can create a _query object_ or at least put it in a method